src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -0,0 +1,25 @@
+--Made by Wilma456
+local args = {...}
+
+if not args[1] then
+  print("Usage: webrun <url> <args>")
+  return
+end
+
+if not http then
+  print("Please enable http in the config")
+  return
+end
+
+local test,text = http.checkURL(args[1])
+if test == false then
+  print("[Error]"..text)
+  return
+end
+
+local data = http.get(args[1])
+local program = data.readAll()
```

Comment:
If `http.get` fails, `data` will be `nil` and the program will crash. It should check whether `http.get` succeeded and fail gracefully if not.